app/controllers/users/phone_setup_controller.rb

```diff
@@ -27,6 +27,8 @@ def create
 
       if result.success?
         handle_create_success(@new_phone_form.phone)
+      elsif recoverable_recaptcha_error?(result)
+        render :spam_protection, locals: { two_factor_options_path: two_factor_options_path }
```

Comment:
just thinking out loud, if this new template is so different from existing stuff, should we add a new controller for it? so we redirect to `users/phone_setup/spam_production#index` or something?

counterpoint, would we be able to easily link back to the phone setup from that new controller if we did?

Reply:
Yeah, I noodled on a few different approaches here. One key thing that is hard to do with a lot of different implementation options is how to maintain form values between the initial submission and when the user successfully passes the checkbox. As implemented here without a redirect, those form values carry over automatically. With a redirect, we'd have to find some other way to save those values (e.g. session).